gridt/src/app/app.component.html

```diff
@@ -15,12 +15,13 @@
               </ion-label>
             </ion-item>
           </ion-menu-toggle>
-        </ion-list>
-        
-        <ion-button expand="full">
-          Logout
-        </ion-button>
+        </ion-list>      
       </ion-content>
+      <ion-footer>
+        <ion-button expand="full" (click)="auth.logout()" href="/">
```

Comment:
It might be interesting to make use of a nice icon here as well. I would recommend `<ion-icon name="exit-outline"></ion-icon>`.

Reply:
Also, for not wanting to attract too much attention to something we wish user use the least, maybe consider using color="light", to make it less attention drawing.